### ISSUE TITLE
Netbeans output window

### ISSUE THREAD

**Issue by rjgamer** [Freyskeyd/gulp-prompt#59]
Strange behaviour in Netbeans output window: The input line for the answer appears first. After entering the anwser, the question of the prompt appears as second line. But the order should actually be reversed. In Powershell the order of the two lines is correct (first the question, second the input).

**Comment by shannonlal** [comment]
@rjgamer Is this issue only on Windows?  Does it only happen in netbeans?  If you use the command line does it happen?  I currently have access to a Mac so it is difficult to reproduce.  I am jut trying to narrow down what the environment I need to find to reproduce this.

**Comment by shannonlal** [comment]
Just a question to confirm.  If you run this gulp taks from the command line (i.e. on your Windows 10 computer and not in Netbeans) does it work?

**Comment by rjgamer** [comment]
Correct. The prompt is working correctly from the PowerShell and command line tool.

**Comment by rjgamer** [comment]
Could you reproduce my "problem"?
